### DISCUSSION TITLE
不会部署的，详细的部署步骤

### DISCUSSION THREAD

**Discussion by skrleo** [ddiu8081/chatgpt-demo#73] [General]
frist： 非常感谢 developer 奉献了这么一个好的项目， 也为了节省大家的时间，提供一下部署的步骤：
```
# ---- Dependencies ----
FROM node:19.7.0 AS dependencies
WORKDIR /app
COPY package.json /app
RUN npm install

# ---- RUN ----
FROM dependencies AS RUN
WORKDIR /app
COPY . /app
COPY .env.example /app/.env
ENV PORT=3000
VOLUME ["./env"]
CMD [ "npm", "run", "dev", "--", "--port", "3000", "--host", "0.0.0.0"]
```
如果还是不会部署的话 这边帮忙部署环境；只需要提供域名即可；Q:1194191181

**Comment by weknowaive** [comment]
能支持多个key么

**Comment by skrleo** [comment]
> 能支持多个key么

不支持

**Comment by SuperRonin** [comment]
国内手机号注册的vercel, deploy成功后，发现消息接口超时，是因为墙？需要注册US地区的账号重新deploy吗？

**Comment by skrleo** [comment]
> 国内手机号注册的vercel, deploy成功后，发现消息接口超时，是因为墙？需要注册US地区的账号重新deploy吗？
我也是国内手机号注册的，deploy成功也没有消息接口超时的情况，估计是你的env没有配置chatgpt 接口

**Comment by hieiip1203** [comment]
![image](https://user-images.githubusercontent.com/125166747/222871747-6e4cfcc5-1f9e-455d-8e3f-904c51259611.png) 项目已经配置好了，但是一直node无法启动 什么意思

**Comment by Lucoding00** [comment]
> > 国内手机号注册的vercel, deploy成功后，发现消息接口超时，是因为墙？需要注册US地区的账号重新deploy吗？
> > 我也是国内手机号注册的，deploy成功也没有消息接口超时的情况，估计是你的env没有配置chatgpt 接口

怎么配置chatgpt接口，我这边有clash for windows

**Comment by clearcodecn** [comment]
run dev 也算部署吗?
